### ISSUE TITLE
Train a random forest classifier

### ISSUE THREAD

**Issue by covuworie** [covuworie/nobel-physics-prizes#80]
Train a random forest classifier using the original features and the topics features with and without the sample weights. Select the following hyperparameters based on maximum MCC on the validation set:

- The number of trees in the forest `n_estimators`.
- The number of features `max_features` to consider when looking for the best split.
- `class_weight`, the weights associated with the classes.
